src/core/services/dom.observe.d.ts

```diff
@@ -0,0 +1,5 @@
+export declare function children(element: object): object;
```

Comment:
DomElements not object, also sometimes Object is used sometimes object choose one please